src/data-sources/DataSourceMetaTags.tsx

```diff
@@ -0,0 +1,66 @@
+import { Icon } from '@wordpress/components';
+import { chevronRightSmall } from '@wordpress/icons';
+
+import { DataSourceConfig } from '@/data-sources/types';
+import './DataSourceList.scss';
+
+interface DataSourceMetaTagsProps {
+	source: DataSourceConfig;
+}
+
+interface DataSourceMetaTag {
+	key: string;
+	primaryValue: string;
+	secondaryValue?: string;
+}
+
+const DataSourceDescriptor = ( props: DataSourceMetaTagsProps ) => {
+	let tag: DataSourceMetaTag | undefined;
+
+	switch ( props.source.service ) {
+		case 'airtable':
+			tag = {
+				key: 'base',
+				primaryValue: props.source.service_config.base?.name,
+				secondaryValue: props.source.service_config.tables?.[ 0 ]?.name,
+			};
+			break;
+		case 'shopify':
+			tag = { key: 'store', primaryValue: props.source.service_config.store_name };
+			break;
+		case 'google-sheets':
+			tag = {
+				key: 'spreadsheet',
+				primaryValue: props.source.service_config.spreadsheet.name ?? 'Google Sheet',
+				secondaryValue: props.source.service_config.sheet.name,
+			};
+			break;
+	}
+
+	if ( ! tag ) {
+		return null;
+	}
+
+	return (
+		<span key={ tag.key } className="data-source-meta">
+			{ tag.primaryValue }
+			{ tag.secondaryValue && (
+				<>
+					<Icon icon={ chevronRightSmall } style={ { fill: '#949494', verticalAlign: 'middle' } } />
+					{ tag.secondaryValue }
+				</>
+			) }
+		</span>
+	);
+};
+
+const DataSourceMetaTags = ( props: DataSourceMetaTagsProps ) => {
+	return (
+		<>
+			{ ! props.source.uuid && <span className="data-source-badge">Code</span> }
```

Comment:
What do you think about a `Tooltip` (from `@wordpress/components`) here with some more information?

```suggestion
			{ ! props.source.uuid && <Tooltip text={ __( 'This source is configured in code', 'remote-data-blocks' ) }><span className="data-source-badge">Code</span></Tooltip> }
```

The result looks like this:

![Screenshot 2024-12-26 at 2 27 09 PM](https://github.com/user-attachments/assets/416a1b1f-787d-44f8-8582-531b40e9a624)

Reply:
I dig it.

Reply:
Done in https://github.com/Automattic/remote-data-blocks/pull/264/commits/86384cbb06325417fd7c8391420d5460c0f269d6